### PR TITLE
Add Brazilian Portuguese localization

### PR DESCRIPTION
**Pull request by Pixelnando**
Translated the mod. If possible, make it so prefixes are as follow: "Item (Prefix)"

### CODE REVIEW

**Comment by direwolf420** [comment]
Big thanks, as mentioned [here](https://github.com/SamsonAllen13/ClickerClass#localization), remove the `# ` infront of every line you translated so it actually applies the translation

File: Localization/pt-BR.lang [hunk 1]
```diff
@@ -60,361 +60,361 @@
...
 
-# ItemName.ClickingGlove=Clicking Glove
-# ItemTooltip.ClickingGlove={$Mods.ClickerClass.Common.Tooltips.ClickingGloveEvery} 3 seconds
+# ItemName.ClickingGlove=Luva de Clicar
+# ItemTooltip.ClickingGlove={$Mods.ClickerClass.Common.Tooltips.ClickingGloveEvery} 3 segundos
 
-# ItemName.Cookie=Cookie
-# ItemTooltip.Cookie=While equipped, cookies will periodically spawn within your clicker radius\nClick the cookie to gain bonus clicker damage, radius, and life regeneration
+# ItemName.Cookie=Biscoito
+# ItemTooltip.Cookie=Enquanto equipado, biscoitos serão gerados periodicamente dentro do seu raio de clique\nClique o biscoito para ganhar Click the cookie to gain bonus clicker damage, radius, and life regeneration
```
Comment (direwolf420) [inline Localization/pt-BR.lang]: leftover ` Click the cookie to gain bonus clicker damage, radius, and life regeneration` at the end

File: Localization/pt-BR.lang [hunk 2]
```diff
@@ -60,361 +60,361 @@
...
-# ItemTooltip.MiceMask={$Mods.ClickerClass.Common.Tooltips.IncreasesClickDamage} by 4%\n{$Mods.ClickerClass.Common.Tooltips.IncreasesClickCritChance} by 6%
+# ItemName.MiceMask=Máscara do Mouse
+# ItemTooltip.MiceMask={$Mods.ClickerClass.Common.Tooltips.IncreasesClickDamage} em 4%\n{$Mods.ClickerClass.Common.Tooltips.IncreasesClickCritChance} em 6%
 
-# ItemName.MiceSuit=Mice Suit
-# ItemTooltip.MiceSuit={$Mods.ClickerClass.Common.Tooltips.IncreasesClickDamage} by 10%\n{$Mods.ClickerClass.Common.Tooltips.IncreasesBaseClickRadius} by 25%
-# SetBonus.Mice=Right clicking a position within your clicker radius will teleport you to it
+# ItemName.MiceSuit=Armadura do Mouse
+# ItemTooltip.MiceSuit={$Mods.ClickerClass.Common.Tooltips.IncreasesClickDamage} by 10%\n{$Mods.ClickerClass.Common.Tooltips.IncreasesBaseClickRadius} em 25%
+# SetBonus.Mice=Clicar com o botão direito em uma posição dentro de seu raio de clique teleportará você para lá
```
Comment (direwolf420) [inline Localization/pt-BR.lang]: `by` untranslated

**Comment by Pixelnando** [comment]
Thanks, fixed both of those issues.

**Pull request merged**